lib/charms/mongodb/v1/mongodb_tls.py

```diff
@@ -235,7 +256,7 @@ def _on_certificate_available(self, event: CertificateAvailableEvent) -> None:
             # clear waiting status if db service is ready
             self.charm.status.set_and_share_status(ActiveStatus())
 
-    def waiting_for_certs(self):
+    def waiting_for_both_certs(self):
```

Comment:
nit:
```suggestion
    def is_waiting_for_both_certs(self) -> bool:
```

Reply:
unfortunately this lib is owned by the VM mongodb charm. Is this a blocker?